src/sonic-yang-mgmt/tests/libyang-python-tests/sonic_config_data.json

```diff
@@ -0,0 +1,276 @@
+{
+	"test-sonic-vlan:sonic-vlan": {
+		"test-sonic-vlan:VLAN_INTERFACE": {
+			"VLAN_INTERFACE_LIST": [{
+					"vlanid": 111,
+					"ip-prefix": "2000:f500:45:6709::1/64",
+					"scope": "global",
+					"family": "IPv6"
+				},
+				{
+					"vlanid": 111,
+					"ip-prefix": "10.1.1.65/26",
+					"scope": "global",
+					"family": "IPv4"
+				},
+				{
+					"vlanid": 111,
+					"ip-prefix": "fe80::1/10",
+					"scope": "local",
+					"family": "IPv6"
+				},
+				{
+					"vlanid": 555,
+					"ip-prefix": "2000:f500:41:4e9::1/64",
+					"scope": "global",
+					"family": "IPv6"
+				},
+				{
+					"vlanid": 555,
+					"ip-prefix": "10.1.5.65/26",
+					"scope": "global",
+					"family": "IPv4"
+				},
+				{
+					"vlanid": 555,
+					"ip-prefix": "fe80::1/10",
+					"scope": "local",
+					"family": "IPv6"
+				}
+			]
+		},
+
+		"test-sonic-vlan:VLAN": {
+			"VLAN_LIST": [{
+					"vlanid": 111,
+					"description": "server_vlan",
+					"dhcp_servers": [
+						"10.1.7.116"
+					],
+					"mtu": "9216",
+					"admin_status": "up",
+					"members": [
+						"Ethernet8",
+						"Ethernet3",
+						"Ethernet0",
+						"Ethernet1",
+						"Ethernet6",
+						"Ethernet4",
+						"Ethernet5",
+						"Ethernet9",
+						"Ethernet2"
+					]
+				},
+				{
+					"vlanid": 555,
+					"description": "ipmi_vlan",
+					"dhcp_servers": [
+						"10.1.7.116"
+					],
+					"mtu": "9216",
+					"admin_status": "up",
+					"members": [
+						"Ethernet9",
+						"Ethernet2",
+						"Ethernet8"
+					]
+				}
+			]
+		},
+
+		"test-sonic-vlan:VLAN_MEMBER": {
+			"VLAN_MEMBER_LIST": [{
+					"vlanid": 111,
+					"port": "Ethernet0",
+					"tagging_mode": "tagged"
+				},
+				{
+					"vlanid": 111,
+					"port": "Ethernet1",
+					"tagging_mode": "tagged"
+				},
+				{
+					"vlanid": 111,
+					"port": "Ethernet2",
+					"tagging_mode": "tagged"
+				},
+				{
+					"vlanid": 111,
+					"port": "Ethernet3",
+					"tagging_mode": "tagged"
+				},
+				{
+					"vlanid": 111,
+					"port": "Ethernet4",
+					"tagging_mode": "tagged"
+				},
+				{
+					"vlanid": 111,
+					"port": "Ethernet5",
+					"tagging_mode": "tagged"
+				},
+				{
+					"vlanid": 111,
+					"port": "Ethernet6",
+					"tagging_mode": "tagged"
+				}
+			]
+		}
+	},
+	"test-sonic-port:sonic-port": {
+		"test-sonic-port:PORT": {
+			"PORT_LIST": [{
+					"port_name": "Ethernet0",
+					"alias": "eth0",
```

Comment:
alias: etp0

Reply:
This is test data for sonic_yang_mgmt library, since we do not have any constraint in test yang models, we can provide any data. I can write 'etp0', but it will not matter in this context.